service/test/BatchServerTest.cpp

```diff
@@ -177,6 +205,57 @@ TEST_F(BatchServerTest, stop_batch)
     EXPECT_FALSE(m_batch_server->is_active());
 }
 
+/**
+ * @throws geopm Exception to simulate failure of sigqueue(2) system call in POSIXSignalImp::sig_queue()
+ */
+ACTION(sig_queue_SIGTERM)
+{
+    throw geopm::Exception(
+        "POSIXSignal(): POSIX signal function call "
+        "sigqueue()"
+        " returned an error",
+        EINVAL,
+        __FILE__,
+        __LINE__
+    );
+}
+
+/**
+ * @test First check that BatchServerImp::stop_batch() throws an exception if it is called from child process.
+ *       Then check that activate the catch block in BatchServerImp::stop_batch() by failure of POSIXSignalImp::sig_queue()
+ */
+TEST_F(BatchServerTest, stop_batch_exception)
+{
+    GEOPM_EXPECT_THROW_MESSAGE(
+        m_batch_server_child_process->stop_batch(),
+        GEOPM_ERROR_RUNTIME,
+        "BatchServerImp::stop_batch(): must be called from parent process, not child process"
+    );
```

Comment:
```suggestion
    GEOPM_EXPECT_THROW_MESSAGE(m_batch_server_child_process->stop_batch(),
                               GEOPM_ERROR_RUNTIME, 
                               "BatchServerImp::stop_batch(): must be called from parent process, not child process");
```

Reply:
Please use this style for spacing and line termination for function calls.  Please let me know if you have questions.

There are many calls to `GEOPM_EXPECT_THROW_MESSAGE` in this file.  Please fix all of them.